tagcompare/capture.py

```diff
@@ -14,11 +17,164 @@
 LOGGER = logger.Logger(name="capture", writefile=True).get()
 
 
-def __capture_tags_for_configs(cids, pathbuilder,
-                               configs,
-                               tagsizes=settings.DEFAULT.tagsizes,
-                               tagtypes=settings.DEFAULT.tagtypes,
-                               capture_existing=False):
+class TagCapture(object):
+    """TagCapture uses webdriver to capture tags for campaigns"""
+
+    def __init__(self, configname, driver, caps=None):
+        self._configname = configname
+        self._driver = driver
+        self._caps = caps
+
+    def stop(self):
```

Comment:
You might consider calling this `close()`, which is something of a convention and would allow `TagCompare` instances to be used with `contextlib.closing`, for example.